backend/FwLite/MiniLcm/Filtering/EntryFilter.cs

```diff
@@ -29,12 +29,18 @@ public static GridifyMapper<T> NewMapper<T>(EntryFilterMapProvider<T> provider)
     }
 
     //used by the database for json columns which are lists, we want to treat null as an empty list
-    public static object ConvertNullToEmptyList<T>(string value)
+    public static object NormalizeEmptyToEmptyList<T>(string value)
     {
         if (value is "null" or "" or "[]") return new List<T>();
         throw new Exception($"Invalid value {value} for {typeof(T).Name}");
     }
 
+    public static object NormalizeEmptyToNullString<T>(string value)
+    {
+        if (value is "null" or "" or "[]") return "null";
```

Comment:
does this actually work? Previously I was returning `null` not `"null"`

Reply:
I don't think we were ever rerturning `null`. You set the converter to `null`, i.e. turned of any conversions, which would leave `"null"` as is.

Reply:
But, this code is tested by e.g.
```
    [Fact]
    public async Task CanFilterToMissingSemanticDomainsWithEmptyArray()
    {
        var results = await Api.GetEntries(new(Filter: new() { GridifyFilter = "Senses.SemanticDomains=[]" })).ToArrayAsync();
        results.Select(e => e.LexemeForm["en"]).Should().BeEquivalentTo(Peach);
    }
```